scheduler/Dockerfile.scheduler

```diff
@@ -1,13 +1,13 @@
-FROM golang:1.19.3-alpine as builder
+FROM golang:1.19.3-buster as builder
 
 WORKDIR /build
 COPY . .
 
 # Build the binary
-RUN apk add --no-cache make
+RUN apt-get install make
 RUN make -C scheduler -o test-go build-scheduler
 
-# Copy binary
+# Kafka dependencies necessitate leaving CGo enabled and using a base image with C dependencies
```

Comment:
:question: I'm probably missing something, but why does the _scheduler_ need to have any interaction with Kafka?

Reply:
As the Kafka Config contains the prefix and that has a reference to the Kafka library.

Reply:
future PR to allow conversion of JSON to KafkaConfig to separate package so Scheduler does not need Kafka library